orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/cache/ClouddriverClearAltTablespaceTask.java

```diff
@@ -48,18 +47,21 @@ public TaskResult execute(@Nonnull StageExecution stage) {
           result.getOrDefault("tables", emptyList()));
 
       return TaskResult.builder(ExecutionStatus.SUCCEEDED).context(result).build();
-    } catch (RetrofitError e) {
-      Map<String, Object> output = new HashMap<>();
-      List<String> errors = new ArrayList<>();
+    } catch (SpinnakerHttpException e) {
+      output = new HashMap<>();
 
-      if (e.getResponse() != null && e.getResponse().getBody() != null) {
-        String error = new String(((TypedByteArray) e.getResponse().getBody()).getBytes());
-        log.error("Failed clearing clouddriver table namespace: {}", error, e);
-        errors.add(error);
+      if (e.getResponseBody() != null) {
+        Map<String, Object> errorBody = e.getResponseBody();
+        log.error("Failed clearing clouddriver table namespace: {}", errorBody, e);
+        output.put("errors", errorBody);
```

Comment:
Before this PR the errors property contained the response body as a String.  The way the code is now, the errors property is either a String or a Map.  I think this makes things tough on consumers.  We're stuck making some kind of behavior change since we no longer have access to the entire response body.  Seems like it's sufficient/less disruptive to use the exception message.  At least the type is consistently a String, which is what it was before.

As I look at the [underlying clouddriver endpoint](https://github.com/spinnaker/clouddriver/blob/44124e218892abce09a0782b826913e62385a7d9/cats/cats-sql/src/main/kotlin/com/netflix/spinnaker/cats/sql/controllers/CatsSqlAdminController.kt#L36) I don't see any special exception handling, so I expect all the interesting info is in the exception message.

Reply:
FYI - Before this PR the errors property contained either a String or List of Strings , it was not consistent even before.

Reply:
FYI - Before this PR the errors property contained List of Strings. However , I got the context . 
Will do the required changes to make the property values consistent

Reply:
To avoid the breakage in consumers of the pipeline execution context, I have modified the errors property to hold List of Strings which retains the state like it was before.